web/modules/custom/pineapple_mail/pineapple_mail.module

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @file
+ * Contains pineapple_mail.module.
+ */
+
+use Drupal\Component\Utility\Html;
+
+/**
+ * Implements hook_mail().
+ */
+function pineapple_mail_mail($key, &$message, $params) {
+  $options = array(
+    'langcode' => $message['langcode'],
+  );
+  switch ($key) {
+    case 'node_insert':
+      $message['from'] = \Drupal::config('system.site')->get('mail');
+      $message['subject'] = t('Subject: @title', array('@title' => $params['title']), $options);
+      $message['body'][] = Html::escape($params['message']);
+      break;
+  }
+}
+
+function pineapple_mail_ready_to_review(Drupal\Core\Entity\EntityInterface $entity) {
+  $mailManager = \Drupal::service('plugin.manager.mail');
+  $module = 'pineapple_mail';
+  $key = 'node_insert';
+  $to = \Drupal::config('system.site')->get('mail'); // site email
+  $entity_link = $entity->toUrl()->toString();
+  $entity_title = $entity->label();
+  $params['message'] = 'The user ' .\Drupal::currentUser()->getAccountName(). ' indicated that ' .$entity_title. ' is ready for review. View it here: www.easyrecipedelight.com' .$entity_link;
+  $params['title'] = $entity_title . ' is ready to review';
+  $langcode = \Drupal::currentUser()->getPreferredLangcode();
+  $send = true;
+
+  $result = $mailManager->mail($module, $key, $to, $langcode, $params, NULL, $send);
+  if ($result['result'] !== true) {
```

Comment:
Can you please add in the drupal logger back in as it was with the original source code, It'll be useful for debugging purposes to have the success and failures in the log page.

Reply:
Done.